doc/source/io.rst

```diff
@@ -1994,6 +1994,18 @@ into a flat table.
 .. ipython:: python
 
    from pandas.io.json import json_normalize
+   data = [
+             {'id': 1, 'name': {'first': 'Coleen', 'last': 'Volk'}},
+             {'name': {'given': 'Mose', 'family': 'Regner'}},
+             {'id': 2, 'name': 'Faye Raker'}
+          ]
+   json_normalize(data)
+
+.. _io.jsonl:
+
+.. ipython:: python
+
+   from pandas.io.json import json_normalize
```

Comment:
It's OK to keep it as separate ipython blocks, if you want to have it render as two separate blocks (which can give some more structure to the examples). In this a case that is OK because it are two different examples, I think.

You don't need to import json_normalize the second time